android/app/build.gradle

```diff
@@ -153,46 +201,46 @@ dependencies {
     *    REACT NATIVE FIREBASE
     * ---------------------------- */
 
-    // Library as defined in settings.gradle
-    implementation project(path: ':react-native-firebase')
+    // Firebase bom setup
+    implementation platform("com.google.firebase:firebase-bom:22.2.0")
 
     // Required dependencies
     //noinspection GradleCompatible
-    implementation "com.google.firebase:firebase-core:16.0.9"
-    implementation "com.google.android.gms:play-services-base:16.1.0"
+    implementation "com.google.firebase:firebase-core"
+    implementation "com.google.android.gms:play-services-base:17.1.0"
```

Comment:
I just double-checked my work project, I no longer have this in there. It must be a transitive dependency, because I build all day long and it works without it. Could you try removing it? That would be the last remaining non-BOM versioned thing, which is a real simplifying factor for people I think

Reply:
Remove `implementation "com.google.android.gms:play-services-base:17.1.0"` or just the version number appended to the end?

Reply:
The whole thing - I don't have that dependency at all in my android/app/build.gradle any more

Reply:
:crossed_fingers:

Reply:
Tested and working without. Pushed change!